### PR TITLE
add support for Devuan (fully Debian compatible)

### PR DESCRIPTION
**Pull request by AlisterAmo**
Devuan 1 is a fully compliant Debian 8 Jessie, with the only modification of the init system (it gets rid of systemD).
100% working at the time of submitting.

### CODE REVIEW

File: scriptmodules/system.sh [hunk 1]
```diff
@@ -87,6 +87,12 @@ function get_os_version() {
                 __platform_flags+=" xbian"
             fi
             ;;
+         Devuan)
+            # Devuan 1 === Debian 8 Jessie, with the only modification
+            # of a different init system (it gets rid of systemD)
+            # so any Devuan version meets the requeriments
+            foo=bar
```
Comment (joolswills) [inline scriptmodules/system.sh]: is this here as a "noop" ? This PR needs a little work - what is `__os_debian_ver` set to in this case ?

Comment (joolswills) [inline scriptmodules/system.sh]: Sorry, i mean `__os_release`

**Comment by AlisterAmo** [comment]
Hi again and sorry for the delay.
* You mean the foo=bar? A noop, yes.
* devuan __os_release is "1", hence the comments I wrote expecting to clarify that. Devuan 1 == Debian 8

Cheers!

**Comment by joolswills** [comment]
It was `1.0` not 1 which is why I was clarifying but I didn't use it in the end.

**Comment by joolswills** [comment]
ps. can use `:` for a no-op rather than setting a global variable.

**Comment by joolswills** [comment]
(I don't think it actually needed a no-op in this case anyway)

**Comment by AlisterAmo** [comment]
Fine!
My intention was more to show that possibility of adding support to devuan with no effort, maybe a small comment or issue somewhere else had been better idea instead of documenting that in a request. And sorry for the bad habits writing shell code, too! My bad altogether! :-P
Cheers, and thanks for the great job.
Alex

**Comment by joolswills** [comment]
:) Thanks for the heads up - I didn't test it much on Devuan - there may be an issue with a few of things that are linked to the init.d system (eg the autostart functionality).

**Pull request closed without merging**